docs/documentation/maya/simple_setup.md

```diff
@@ -1,37 +1,62 @@
 # A Simple Setup
 
-This page will be focused on displaying the process of creating and setting up each deformer in maya showing the process step by step.
+This page is dedicated to explain, step by step, the process of creating and setting every Adonis deformer in Maya. The scenarios set up here will be basic cases with the minimum configuration to start observing expected results in each case.
 
 ## AdnSkin simple setup
 
-For AdnSkin we will start with a scene with the following elements:
+To create a basic scenario using the AdnSkin deformer we will start with a scene with the following elements:
 
   - A target mesh with animation.
   - A simulated mesh (without animation).
 
 The AdnSkin deformer will be applied to the simulated mesh.
 
+<figure>
+  <img src="images/setup_skin_0.png"> 
+  <figcaption>Figure 1: Basic set up for skin simulations.</figcaption>
+</figure>
+
 ### 1. Creating the deformer
 
 To create the deformer we will select the target mesh and then the skin mesh (simulated mesh). Then we will press the ![adnMuscle](images/adn_skin.png) shelf button or go to *AdonisFX menu > Skin*.
```

Comment:
Following the suggestions from skin we are only using ** for the button. For example here it would be 
```suggestion
To create the deformer we will select the target mesh and then the skin mesh (simulated mesh). Then we will press the ![adnMuscle](images/adn_skin.png) shelf button or go to AdonisFX menu > *Skin*.
```

Please review this for the whole document